keyboards/b_sides/rev41lp/readme.md

```diff
@@ -0,0 +1,15 @@
+# rev39lp
```

Comment:
We mean Rev41lp here, correct?

Reply:
Honestly, I haven't truly settled on a brand for my stuff just yet...  
I lean toward 'b_sides' over 'cyril279', but I don't have strong feelings on the matter.